docs/markdown/flexible-plots.md

```diff
@@ -0,0 +1,35 @@
+## Flexible plots
+
+By default, XYPlot requires a width and a height. There are times, however, when you'd like your chart to take all the space it can. For these cases, React-vis provides a helper function to make one or both dimensions of your chart flexible.
```

Comment:
docs note: we should probably decide on a consistant spelling of react-vis, eg is React-Vis/React Vis/react-vis/etc